runpod/serverless/modules/rp_job.py

```diff
@@ -29,9 +45,10 @@ async def get_job(session: ClientSession, retry=True) -> Optional[Dict[str, Any]
     Note: Retry True just for ease of, if testing improved this can be removed.
     """
     next_job = None
+
     while next_job is None:
         try:
-            async with session.get(JOB_GET_URL) as response:
+            async with session.get(_job_get_url()) as response: # pylint: disable=line-too-long
```

Comment:
`# pylint: disable=line-too-long` should only be used if absolutely necessary, otherwise refactor. 

However, since you refactored this does not appear to be necessary now.

Reply:
Agreed, that's a relic prior to refactoring.